### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @tomastoloza

### CODE REVIEW

**Comment by tomastoloza** [comment]
@faloi si querés corregir...

File: README.md [hunk 1]
```diff
@@ -23,3 +23,6 @@ Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archi
 ![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
 ![Desde un archivo local](./images/ejemplo-uml.png)
+
+**Diagramas de clase**
+![Diagrama](./images/uml-documentando-wollok.png)
```
Comment (faloi) [inline README.md:28]: Ok. No lo dije explícitamente, pero me hubiera gustado que se vean los tipos acá.

File: src/elementos.wlk [hunk 2]
```diff
@@ -17,12 +18,14 @@ class Mascota {
 
 	var property nivelSalud
 
+    // No devuelve nada, recibe parametro de cucarachas, de mosquitos, de pulgas o de garrapatas
 	method fueAtacado(plaga) {
```
Comment (faloi) [inline src/elementos.wlk:22]: Bien. Genéricamente podríamos decir que recibe algo del tipo `Plaga`.

File: src/plagas.wlk [hunk 3]
```diff
@@ -1,8 +1,10 @@
 class Plaga {
 	var property poblacion
-	
+
+	// No devuelve nada
```
Comment (faloi) [inline src/plagas.wlk:4]: Ojo, devuelve `boolean`. La sintaxis `method ... = ...` es una forma que tiene Wollok para definir en una sola línea un método que devuelve algo.

**Comment by tomastoloza** [comment]
> Hola @tomastoloza, muy buena entrega. Mirá los comentarios.

@faloi, podríamos decir que la entrega está ok o se necesita corregir?

**Comment by faloi** [comment]
> > Hola @tomastoloza, muy buena entrega. Mirá los comentarios.
> 
> @faloi, podríamos decir que la entrega está ok o se necesita corregir?

Está ok. No vamos a hacer reentregas ni correcciones en los ejercicios semanales.

**Pull request closed without merging**